projects/kit/components/input-phone-international/tokens/countries-masks.ts

```diff
@@ -42,7 +42,7 @@ export const TUI_COUNTRIES_MASKS: InjectionToken<Record<TuiCountryIsoCode, strin
             [TuiCountryIsoCode.CF]: `+236-##-##-####`,
             [TuiCountryIsoCode.CG]: `+242##-###-####`,
             [TuiCountryIsoCode.CH]: `+41##-###-####`,
-            [TuiCountryIsoCode.CI]: `+225##-###-###`,
+            [TuiCountryIsoCode.CI]: `+225##-##-###-###`,
```

Comment:
`Costa de Marfil` format is  `+225 XX XX XXX XXX` +225 and  10  - https://en.wikipedia.org/wiki/Telephone_numbers_in_Ivory_Coast